contrib/npm/README.md

```diff
@@ -0,0 +1,77 @@
+# Hurl
+
+Hurl is a command line tool that runs <b>HTTP requests</b> defined in a simple <b>plain text format</b>.
+
+It can perform requests, capture values and evaluate queries on headers and body response. Hurl is very
+versatile: it can be used for both <b>fetching data</b> and <b>testing HTTP</b> sessions.
+
+
+```hurl
+# Get home:
+GET https://example.net
+
+HTTP/1.1 200
+[Captures]
+csrf_token: xpath "string(//meta[@name='_csrf_token']/@content)"
+
+# Do login!
+POST https://example.net/login?user=toto&password=1234
+X-CSRF-TOKEN: {{csrf_token}}
+
+HTTP/1.1 302
+```
+
+Hurl can run HTTP requests but can also be used to <b>test HTTP responses</b>.
+Different types of queries and predicates are supported, from [XPath] and [JSONPath] on body response,
```

Comment:
The `[...]` syntax here is odd. Did you mean to link to something in Markdown?

Reply:
Ba copy/paste form editing README which uses Markdown reference link, I'll fix it.